web/server/net_logger.go

```diff
@@ -247,7 +247,7 @@ func (nl *netLogger) backgroundWorker() {
 		err := nl.Sync()
 		if err != nil && !errors.Is(err, context.Canceled) {
 			interval = abnormalInterval
-			nl.loggerWithoutNet.Infof("error logging to network: %s", err)
+			nl.loggerWithoutNet.Debugf("error logging to network: %s", err)
```

Comment:
the main issue is that this log is pretty spammy if we are not connected to the internet, but maybe that means we need to be a little smarter about not making so many network logs if we suspect viam.com is unreachable

Reply:
Yeah, that makes sense! One idea is to have a boolean containing whether the last log was successfully sent to the network, and only log this if it's true. Another idea is to leave it the way you've written it, but add in a comment warning folks that this can be super spammy. :)

Reply:
good idea. for the time being, i'm going to keep this at the info level just to reduce how much we are changing in this fix
```suggestion
			nl.loggerWithoutNet.Infof("error logging to network: %s", err)
```